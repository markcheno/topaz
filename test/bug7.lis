#    0: adjs		1
#    5: pushint		35
#   10: setglobal	0
#   15: pushint		0
#   20: pushstring	'string=%s
'
#   36: getglobal	0
#   41: pushint		2
#   46: call		printf
#   51: pop
#   52: halt
