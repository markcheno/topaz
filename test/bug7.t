s = 35

printf("string=%s\n",s)
