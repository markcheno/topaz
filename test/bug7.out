runtime error - wrong type for integer format

